src/common/collection.c

```diff
@@ -615,27 +615,31 @@ const char *dt_collection_name(dt_collection_properties_t prop)
   char *col_name = NULL;
   switch(prop)
   {
-    case DT_COLLECTION_PROP_FILMROLL:     return _("film roll");
-    case DT_COLLECTION_PROP_FOLDERS:      return _("folders");
-    case DT_COLLECTION_PROP_CAMERA:       return _("camera");
-    case DT_COLLECTION_PROP_TAG:          return _("tag");
-    case DT_COLLECTION_PROP_DAY:          return _("date");
-    case DT_COLLECTION_PROP_TIME:         return _("time");
-    case DT_COLLECTION_PROP_HISTORY:      return _("history");
-    case DT_COLLECTION_PROP_COLORLABEL:   return _("color label");
-    case DT_COLLECTION_PROP_LENS:         return _("lens");
-    case DT_COLLECTION_PROP_FOCAL_LENGTH: return _("focal length");
-    case DT_COLLECTION_PROP_ISO:          return _("ISO");
-    case DT_COLLECTION_PROP_APERTURE:     return _("aperture");
-    case DT_COLLECTION_PROP_EXPOSURE:     return _("exposure");
-    case DT_COLLECTION_PROP_ASPECT_RATIO: return _("aspect ratio");
-    case DT_COLLECTION_PROP_FILENAME:     return _("filename");
-    case DT_COLLECTION_PROP_GEOTAGGING:   return _("geotagging");
-    case DT_COLLECTION_PROP_GROUPING:     return _("grouping");
-    case DT_COLLECTION_PROP_LOCAL_COPY:   return _("local copy");
-    case DT_COLLECTION_PROP_MODULE:       return _("module");
-    case DT_COLLECTION_PROP_ORDER:        return _("module order");
-    case DT_COLLECTION_PROP_LAST:         return NULL;
+    case DT_COLLECTION_PROP_FILMROLL:         return _("film roll");
+    case DT_COLLECTION_PROP_FOLDERS:          return _("folders");
+    case DT_COLLECTION_PROP_CAMERA:           return _("camera");
+    case DT_COLLECTION_PROP_TAG:              return _("tag");
+    case DT_COLLECTION_PROP_DAY:              return _("date");
+    case DT_COLLECTION_PROP_TIME:             return _("time");
+    case DT_COLLECTION_PROP_HISTORY:          return _("history");
+    case DT_COLLECTION_PROP_COLORLABEL:       return _("color label");
+    case DT_COLLECTION_PROP_LENS:             return _("lens");
+    case DT_COLLECTION_PROP_FOCAL_LENGTH:     return _("focal length");
+    case DT_COLLECTION_PROP_ISO:              return _("ISO");
+    case DT_COLLECTION_PROP_APERTURE:         return _("aperture");
+    case DT_COLLECTION_PROP_EXPOSURE:         return _("exposure");
+    case DT_COLLECTION_PROP_ASPECT_RATIO:     return _("aspect ratio");
+    case DT_COLLECTION_PROP_FILENAME:         return _("filename");
+    case DT_COLLECTION_PROP_GEOTAGGING:       return _("geotagging");
+    case DT_COLLECTION_PROP_GROUPING:         return _("grouping");
+    case DT_COLLECTION_PROP_LOCAL_COPY:       return _("local copy");
+    case DT_COLLECTION_PROP_MODULE:           return _("module");
+    case DT_COLLECTION_PROP_ORDER:            return _("module order");
+    case DT_COLLECTION_PROP_IMPORT_TIMESTAMP: return _("import timestamp");
```

Comment:
Please move those 4 new filters after the time above. We try grouping the filters by category.

Reply:
I've adjusted the order of the filters. I have also changed the wording of the date and time filters so that there is no confusion:
date --> date taken
date time --> date-time taken